# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1322344744 0
#      Sat Nov 26 21:59:04 2011 +0000
# Node ID bf209050014f2cdfdec8dd679cd68f9a6a40649a
# Parent  fc6ebd31f40f17b26a2430b0718f8d76b1f2fa13
Use nargout = 0 in graphics callbacks specified as string.

* src/graphics.cc (gh_manager::do_execute_callback): Call eval_string with
nargout set to 0.

diff --git a/src/graphics.cc b/src/graphics.cc
old mode 100644
new mode 100755
--- a/src/graphics.cc
+++ b/src/graphics.cc
@@ -7686,17 +7686,17 @@ gh_manager::do_execute_callback (const g
 
       if (cb.is_function () || cb.is_function_handle ())
         fcn = cb.function_value ();
       else if (cb.is_string ())
         {
           int status;
           std::string s = cb.string_value ();
 
-          eval_string (s, false, status);
+          eval_string (s, false, status, 0);
         }
       else if (cb.is_cell () && cb.length () > 0
                && (cb.rows () == 1 || cb.columns () == 1)
                && (cb.cell_value ()(0).is_function ()
                    || cb.cell_value ()(0).is_function_handle ()))
         {
           Cell c = cb.cell_value ();
 
